assets/src/apps/authoring/components/AdaptivityEditor/ActionMutateEditor.tsx

```diff
@@ -93,6 +93,18 @@ const ActionMutateEditor: React.FC<ActionMutateEditorProps> = (props) => {
     }
   }, [isDirty]);
 
+  // update adding operator if targetType changes from number
+  useEffect(() => {
+    if (targetType !== ('1' as unknown)) {
```

Comment:
`'1' as unknown` is weird, can we parseInt on targetType?

Reply:
fixed.